### DISCUSSION TITLE
Getting error after converting fasttext vectors to spacy

### DISCUSSION THREAD

**Discussion by Sharathmk99** [explosion/spaCy#5988] [Help: Coding & Implementations] [answered]
## How to reproduce the behaviour

I have converted fasttext vectors to spacy using below command
`python -m spacy init-model en /en_fast_text --vectors-loc cc.en.300.vec.gz`
I have downloaded gz file from `https://dl.fbaipublicfiles.com/fasttext/vectors-crawl/cc.en.300.vec.gz`

en_fast_text directory contains below files under vocab
```
 key2row
 lookups.bin
 lookups_extra.bin
 strings.json
 vectors
```

When i tried below code using spacy i get error
`nlp = spacy.load("/en_fast_text")`

Error
`FileNotFoundError: [Errno 2] No such file or directory: 'en_fast_text\\vocab\\lexemes.bin'`

Requesting to please help if i'm missing anything.

## Your Environment

* Operating System: Windows
* Python Version Used: 3.7
* spaCy Version Used: 2.2.4
* Environment Information: conda

**Comment by adrianeboyd** [comment, marked as answer]
Hi, can you double-check that you are running `spacy init-model` with the same spacy version as you're using to load it? It looks like you created the model with spacy v2.3 but then you're trying to load it with v2.2.
